### DISCUSSION TITLE
Assign to user

### DISCUSSION THREAD

**Discussion by krekas** [coderflexx/laravel-ticket#7] [Q&A]
Is there a way to assign ticket to user? Can't see it. Maybe we could add `assigned_to_id` field to `tickets` table?

**Comment by krekas** [comment]
@ousid anything?

**Comment by ousid** [comment]
Hey, @krekas 

Sorry for the late response.

Yes, you can assign the ticket to another user, take a look [here](https://github.com/coderflexx/laravel-ticket#usage)

Since the `Ticket` is a model, you can do the following:

```php
...
use Coderflex\LaravelTicket\Models\Ticket;

public function store()
{
  $user = User::find($userId);

  $ticket = $user->tickets()->create($data);

...

   // or

  $ticket = Ticket::create(array_merge($data, ['user_id' => $userId]);
}
```

You may want to take a look at the [Ticket Structure Table](https://github.com/coderflexx/laravel-ticket#ticket-table-structure) to know more.

**Comment by krekas** [comment]
@ousid but isn't `user_id` for user who created ticket?

**Reply by ousid** [reply]
Not necessary, It may the admin who creates the ticket for the staff, or you want the system to handle and assign the ticket manually for your staff.

The `user_id` is for who creates the ticket. Or I'm missing something? Can you please give more details on what you want to achieve?

**Reply by krekas** [reply]
Manually assign to staff. Like github issue is created by someone and can be assigned to some user.

**Reply by ousid** [reply]
Got it! Yes it's a great idea, if you can make a PR for it, will be much appreciate it!

**Comment by ousid** [comment]
Update on this discussion, the issue in here, is to add the ability to assign users (agents/staff...) to respond to the ticket thread, which the OP, created a PR #8 to resolve the issue.

Thanks, @krekas.
